packages/ui/src/components/support/SupportHeroComponent.tsx

```diff
@@ -0,0 +1,32 @@
+export interface SupportHeroProps {
+  title?: string
+  subtitle?: string
+  backgroundImage?: string
+}
+
+const SupportHeroComponent = ({ backgroundImage, subtitle, title }: SupportHeroProps) => {
+  return (
+    <div className="relative h-screen overflow-hidden">
+      <div
+        className="absolute inset-0 bg-cover bg-center"
+        style={{
+          backgroundImage: `url(${backgroundImage || 'https://sarkac.org/wp-content/uploads/2023/08/dayanisma.png'})`,
+          backgroundPosition: 'bottom',
+          backgroundSize: 'cover',
+          filter: 'brightness(0.7)',
+          opacity: 0.8,
+        }}
```

Comment:
- Use an absolutely positioned `<NextImage />` component for image optimization instead of a background image.
  - Add an additional transparent darker overlay on top of the image to enhance text readability.
- Avoid using static source, title, subtitle etc. directly in the component. You can test the component with a story in Storybook.